cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -107,31 +105,32 @@ <h2 id="interested">Interested in Applying?</h2>
         </ul>
     </section>
 
-    {% for job_applicant_type in page.job_application_types %}
+    {% set usajobs_links = page.usajobs_application_links.order_by(
+      'applicant_type' 
+    ) %}
+    {% for usajobs_link in usajobs_links %}
     <div class="block
                 block__bg
                 block__border">
-        <h4>{{ job_applicant_type.application_type.applicant_type }}</h4>
-        <p>{{ job_applicant_type.application_type.description }}</p>
-
-        {% if job_applicant_type.instructions %}
-            <p>{{ job_applicant_type.instructions }}</p>
-        {% endif %}
-
-        {% if job_applicant_type.usajobs_url %}
-           {% set job_url = job_applicant_type.usajobs_url %}
-        {% elif job_applicant_type.email %}
-           {% set job_url = "mailto:{0}?subject=Application for Position: {1}"
-                            .format(job_applicant_type.email, page.job_title | urlencode) %}
-        {% endif %}
-
-        <p><a class="btn" href="{{ job_applicant_type.usajobs_url }}">Apply now</a></p>
+        <h4>{{ usajobs_link.applicant_type.applicant_type }}</h4>
+        <p>{{ usajobs_link.applicant_type.description }}</p>
+        <p><a class="btn" href="{{ usajobs_link.url }}">Apply now</a></p>
         <p>
             You are about to leave consumerfinance.gov. To submit the
             application, you must go to USAJobs.gov.
         </p>
     </div>
     {% endfor %}
+    {% set email_links = page.email_application_links.order_by('label') %}
+    {% for email_link in email_links %}
+    <div class="block
+                block__bg
+                block__border">
+        <h4>{{ email_link.label }}</h4>
+        {% if email_link.description %}<p>{{ email_link.description }}</p>{% endif %}
+        <p><a class="btn" href="{{ email_link.mailto_link | safe }}">Apply now</a></p>
+    </div>
+    {% endfor %}
```

Comment:
@schaferjh I removed the "You are about to leave consumerfinance.gov" message if the link is a `mailto:` rather than an external link to USAJOBS. Is there other equivalent text that should go here?